Update active-directory-aadconnectsync-attributes-synchronized.md

adamfowlerit opened the pull request:
Fixed the Exchange hybrid writeback attribute names to reflect the correct names in AD.

azurecla commented:
Hi __@meet-bhagdev__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

adamfowlerit committed:
Update active-directory-aadconnectsync-attributes-synchronized.md

Fixed the Exchange hybrid writeback attribute names to reflect the correct names in AD.
